modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -17,6 +17,11 @@ For distributed units (DUs), {rh-rhacm} supports {product-title} deployments tha
 * Create persistent volume custom resources (CR) for database and file system storage.
 * You have installed the OpenShift CLI (`oc`).
 
+[IMPORTANT]
+====
+It is recommended to create a persistent volume resource for image storage. Failure to specify persistent volume storage for images can affect cluster performance.
```

Comment:
```suggestion
Create a persistent volume resource for image storage. Failure to specify persistent volume storage for images can affect cluster performance.
```

Or something to that effect. ISG is unhappy with "It is recommended..." unless there are no alternatives.

Reply:
Updated as suggested.